R/plot_shinygosling.R

```diff
@@ -34,20 +34,16 @@
 #'    am_data("hg38") |>
 #'    filter(uniprot_id == "Q1W6H9") |>
 #'    to_GPos()
-#' 
-#' 
 #'
-#' # Plot the GRanges object
-#' plot_granges(gpos, title = "My GRanges Plot", subtitle = "Custom subtitle")
+#' ## Plot the GRanges object
+#' plot_granges(gpos, mode = "bar", title = "Q1W6H9 track", subtitle 
```

Comment:
format so that the rendered example follows the convention of the package
```r
#' plot_granges(
#'     gpos, mode = "bar", title = "Q1W6H_track",
#'     subtitle = "bar plot example"
#' )
```

Reply:
c65a351